docs/pages/fragments/high-availability.mdx

```diff
@@ -57,8 +57,57 @@ vcluster connect vcluster-1 -n host-namespace-1
 
 # Then execute in a new terminal
 export KUBECONFIG=kubeconfig.yaml
-kubectl get ns
+kubectl get ns  
 ...
 ```
 
+## Enable HA in rootless mode
+
+```
+# Enable HA mode
+enableHA: true
+
+# Scale up syncer replicas
+syncer:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up etcd
+etcd:
+  replicas: 3
+  fsGroup: 12345
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up controller manager
+controller:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up api server
+api:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up DNS server
+coredns:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+```
+You can also refer to [this example](https://github.com/loft-sh/vcluster/issues/674) for HA in rootless mode.
```

Comment:
Please remove this as the new config section essentially is exactly this example and there is no important additional information in the issue